youbet/foundry.toml

```diff
@@ -3,7 +3,18 @@ src = "src"
 out = "out"
 libs = ["lib"]
 
+# for safety validations
```

Comment:
What is this safety validations for? Is there any reference link for me to get more info.